content/_index.html

```diff
@@ -19,8 +19,8 @@
 
 {{% blocks/section title="Integrations" subtitle="Integrate Anchore with your favorite CI/CD Systems, Orchestration Platforms, and more" color="primary" %}}
 
-{{% blocks/feature svg="icons/kubernetes-logo.svg" title="Kubernetes" %}}
-Install Anchore using the [Helm Chart](https://github.com/anchore/anchore-charts/blob/master/stable/anchore-engine) and make admission decisions based on analysis and policy-based evaluations with our [Kubernetes Admission Controller](https://github.com/anchore/kubernetes-admission-controller)
+{{% blocks/feature svg="icons/kubernetes-logo.svg" href="/docs/using/ui_usage/kubernetes/" title="Kubernetes" %}}
```

Comment:
This link will 404 in the built site.
```suggestion
{{% blocks/feature svg="icons/kubernetes-logo.svg" href="/current/docs/using/ui_usage/kubernetes" title="Kubernetes" %}}
```